paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/Expression.java

```diff
@@ -142,7 +148,34 @@ enum ExpressionFunction {
                             referencedField.literals());
                 }),
         CAST((typeMapping, caseSensitive, args) -> cast(args)),
-        NOW((typeMapping, caseSensitive, args) -> new NowExpression());
+        NOW((typeMapping, caseSensitive, args) -> new NowExpression()),
```

Comment:
Can you document these functions to `other_functions.html`?

Reply:
done